apps/backoffice-v2/src/App/App.Router.tsx

```diff
@@ -27,7 +27,7 @@ const routes = [
   caseManagementRoute.addChildren([
     caseManagementIndexRoute,
     individualsRoute.addChildren([individualsIndexRoute, individualRoute]),
-    // companiesRoute.addChildren([companiesIndexRoute]),
+    // businessesRoute.addChildren([businessesIndexRoute]),
```

Comment:
If we keep individuals and businesses as the same route then we no longer need the businesses route code.